.travis.yml

```diff
@@ -16,10 +16,10 @@ install:
   - source activate test-env
   - conda install -q pytest-cov
   - pip install -q flake8 coveralls
-  - python setup.py install
+  - make install
 script:
-  - flake8
+  - make lint
   - python ../q2lint/q2lint.py
```

Comment:
I think you can make the change in this PR and assume the q2lint PR has been merged. The q2lint PR is GTG but we're holding off on merging until the poncho-time PRs are ready for merge and the repos are unfrozen (then the q2lint PR and poncho-time PRs can all be merged at the same time).